ITEventsTests/FullEventPresenterTests/FullEventViewMock.swift

```diff
@@ -0,0 +1,18 @@
+@testable import ITEvents
+
+class FullEventViewMock: IFullEventView {
+    var showMoreEventsButtonCount = 0
+    var similarEventsCount = 0
+    var createdViewsCount = 0
+    var eventViewModel: EventViewModel!
+    
+    func createEventView(with event: EventViewModel, isSimilar: Bool) {
```

Comment:
We don't need to track ```self.similarEventsCount``` the one that passed in ```createShowMoreEventsButton(for eventsCount: Int)```

But we do need to test if our presenter created correct amount of views, so we need to track it this way:
```swift
func createEventView(with event: EventViewModel, isSimilar: Bool) {
   if isSimilar {
      similarEventViewCount++
   } else {
      eventViewCount++
   }
}
```